src/czbenchmarks/tasks/single_cell/perturbation_expression_prediction.py

```diff
@@ -97,12 +96,7 @@ def __init__(
             random_seed (int): Random seed for reproducibility.
         """
         super().__init__(random_seed=random_seed)
-        if metric == "wilcoxon":
-            self.metric_column = "logfoldchange"
-        elif metric == "t-test":
-            self.metric_column = "standardized_mean_diff"
-        else:
-            raise ValueError(f"Metric {metric} not supported")
+        self.metric_column = "logfoldchange" # TODO: logfoldchange only for now
```

Comment:
@polinabinder1 I made one commit of required changes while attempting to test this morning. Unfortunately, still hit bugs when running `load_perturbation_task_input_from_saved_files`. Let's discuss in standup, as I'll need to test this from end to end ASAP.